encfs/BlockFileIO.cpp

```diff
@@ -147,6 +147,10 @@ ssize_t BlockFileIO::read(const IORequest &req) const {
       }
 
       ssize_t readSize = cacheReadOneBlock(blockReq);
+      if (readSize < 0) {
+        result = readSize;
```

Comment:
can you document in the method comment what return value means now?
positive (or zero?) value == success,
negative value (or zero?) == failure?